Jetstream/ModelValue.swift

```diff
@@ -116,7 +116,7 @@ func modelValueIsNillable(type: ModelValueType) -> Bool {
 }
 
 extension String: ModelValue {
-    func equalTo(value: ModelValue) -> Bool { return self == value as String }
+    func equalTo(value: ModelValue) -> Bool { return self == value as! String }
```

Comment:
For sure! It's great to see that the change in syntax uncovers otherwise hidden problems.